code/modules/reqs/supply.dm

```diff
@@ -45,6 +45,8 @@ GLOBAL_LIST_EMPTY_TYPED(exports_types, /datum/supply_export)
 	use_ripples = FALSE
 	var/list/gears = list()
 	var/list/obj/machinery/door/poddoor/railing/railings = list()
+	///The faction of this docking port (aka, on which ship it is located)
+	var/faction = FACTION_TERRAGOV
```

Comment:
explain this

Reply:
It needs a faction to be able to sell. It could be a define in our case, but i'll do another PR after this one refactoring supply lift to be able to place multiple on maps for different factions